spec/prog/vm/vm_host_slice_nexus_spec.rb

```diff
@@ -0,0 +1,245 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require_relative "../../../prog/vm/vm_host_slice_nexus"
+require_relative "../../../prog/vm/host_nexus"
+
+RSpec.describe Prog::Vm::VmHostSliceNexus do
+  subject(:nx) { described_class.new(Strand.create(id: "b231a172-8f56-8b10-bbed-8916ea4e5c28", prog: "Prog::Vm::VmHostSliceNexus", label: "create")) }
+
+  let(:sshable) {
+    Sshable.create_with_id
+  }
+
+  let(:vm_host) {
+    VmHost.create(
+      location: "x",
+      total_cores: 4,
+      used_cores: 1
+    ) { _1.id = sshable.id }
+  }
+
+  let(:vm_host_slice) {
+    VmHostSlice.create_with_id(
```

Comment:
```suggestion
    VmHostSlice.create(
```

We kept `create_with_id` for backwards compatibility, but there is no reason to use it in new code.

Reply:
OK, will change.